scripts/generate_embeddings.py

```diff
@@ -25,6 +24,7 @@ def file_metadata_func(file_path: str) -> Dict:
     """
     docs_url = (
         OCP_VERSION_DOCS_ROOT_URL
```

Comment:
doesn't needs need to be OCP_DOCS_ROOT_URL now?

Reply:
Correct. I fixed it, but the fix went into the second commit on this branch https://github.com/openshift/lightspeed-rag-content/blob/9e72b708dcbef9ae95fb6c7f15e4b1148ceac531/scripts/generate_embeddings.py

Reply:
in the future it would be better to get the change in the right commit (interactive rebase to reorder your commits if needed, or just introduce a 3rd commit and then interactive rebase to squash the 2 related commits)